bash_completions.go

```diff
@@ -387,7 +387,7 @@ __%[1]s_handle_word()
 }
 
 func writePostscript(buf io.StringWriter, name string) {
-	name = strings.Replace(name, ":", "__", -1)
+	name = strings.ReplaceAll(name, ":", "__", )
```

Comment:
The linter is failing.  I hadn't even noticed that the last comma preceding the `-1` was not removed.  This seems to be the case for almost every change 😄 

@WhyNotHugo could you please fix that? Thanks.